### PR TITLE
feat: add listing to each section

### PR DESCRIPTION
**Pull request by tdstein**

### CODE REVIEW

File: 04-content/index.qmd [hunk 1]
```diff
@@ -1,3 +1,16 @@
...
-title: 04. Content
+title: Content
+listing:
+    sort: false
+    template: ../template.ejs
+    contents:
+    - ./02-viewing-content-runtime-settings
+    - ./04-viewing-all-vanity-urls
+    - ./06-finding-public-content
+    - ./06-finding-public-content/images
```
Comment (toph-allen) [inline 04-content/index.qmd:10]:
```suggestion
```

We could probably just remove the `images` subdirectory and change the paths in the content.

**Review by toph-allen — APPROVED** [review]
Do you think it'd be useful to have scripts and/or a `justfile` action to perform these sorts of generative tasks, so they aren't just in your command line history etc.

Since this is still pretty young, I think it's prob a good idea to not make it too fiddly to update chapter names and ordering. Like, right now, to rename a file, you have to update its name in `_quarto.yml` and in the chapter listing order, which is just a hassle.

Hmm, actually, is this to do with the listing sorting bug I see in the escalations Quarto project? If so, hopefully we'll be able to remove the manual listings soon.

Anyway. Approved! Just one pesky subdirectory that made its way in.

**Comment by tdstein** [comment]
> Hmm, actually, is this to do with the listing sorting bug I see in the escalations Quarto project? If so, hopefully we'll be able to remove the manual listings soon.

Which bug are you referring to?

**Comment by tdstein** [comment]
> Do you think it'd be useful to have scripts and/or a `justfile` action to perform these sorts of generative tasks, so they aren't just in your command line history etc.

It could be. I'm hoping there is a simpler way to do this that doesn't require so much manual configuration. I tried editing the template.ejs file to skip rendering files that were empty, allowing for glob expressions in the contents field. But I couldn't get it to work. For some reason, the Quarto metadata doesn't reliably provide information that helps us discern which files are empty.

**Comment by tdstein** [comment]
> > Do you think it'd be useful to have scripts and/or a `justfile` action to perform these sorts of generative tasks, so they aren't just in your command line history etc.
> 
> It could be. I'm hoping there is a simpler way to do this that doesn't require so much manual configuration. I tried editing the template.ejs file to skip rendering files that were empty, allowing for glob expressions in the contents field. But I couldn't get it to work. For some reason, the Quarto metadata doesn't reliably provide information that helps us discern which files are empty.

fwiw, here is my bash command to generate the content listing:

`find ./* -type f -size +0c -exec dirname {} \; | sort -u |  awk '{print "    - "$0}' | sed '1d'`

**Pull request merged**